hackclub.com.yaml

```
@@ -558,7 +558,7 @@ accelerate:
 metabase.accelerate: #alexvd@hackclub
```

On the side, could you correct this email address? Our goal is to be able to pull contact info using regexes.
```suggestion
metabase.accelerate: # alexvd@hackclub.com
```